ddtrace/propagation/http.py

```diff
@@ -750,6 +751,12 @@ def _extract(headers):
                 else:
                     log.debug("no dd list member in tracestate from incoming request: %r", ts)
 
+        if (
```

Comment:
Google sets `traceparent` and `x-cloud-trace-context` headers when our cloud function is triggered, used for Google Cloud Tracing. Because of the is, the tracer sets the `parent_id` of our cloud function trace's root span, thereby causing the span to not show up under the `Trace Root Spans` in Datadog.

I've talked with my team, and we thought it would make sense to drop the span context span_id, in the case that datadog headers aren't set and we are in a google cloud environment.